api/docs/http_catalog.md

```diff
@@ -42,7 +42,7 @@ Three parameters should be provided to the POST or GET:
 
 #### Catalog found
 
-    POST /env/catalog/elmo.mydomain.com
+    POST /puppet/v3/catalog/elmo.mydomain.com?environment=:environment
```

Comment:
Since this is an example, should we have the environment set to `production` instead of `:environment`, since that seems to be the environment based on the example response?